pkg/skuba/actions/node/remove/remove.go

```diff
@@ -70,12 +71,12 @@ func Remove(client clientset.Interface, target string, drainTimeout time.Duratio
 	kubernetes.DrainNode(node, drainTimeout)
 
 	if isControlPlane {
-		fmt.Printf("[remove-node] removing etcd from node %s\n", targetName)
+		klog.Infof("removing etcd from node %s\n", targetName)
 		etcd.RemoveMember(client, node, currentClusterVersion)
 	}
 
 	if err := kubernetes.DisarmKubelet(client, node, currentClusterVersion); err != nil {
-		fmt.Printf("[remove-node] failed disarming kubelet: %v; node could be down, continuing with node removal...\n", err)
+		klog.Errorf("failed disarming kubelet: %v; node could be down, continuing with node removal...\n", err)
```

Comment:
I like this. not sure if it goes with the PR.

Reply:
What is your suggestion?